fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -276,11 +276,7 @@ private void analyzeCommon(Analyzer analyzer) throws AnalysisException {
         // check argument
         argsDef.analyze(analyzer);
         returnType.analyze(analyzer);
-        if (intermediateType != null) {
-            intermediateType.analyze(analyzer);
-        } else {
-            intermediateType = returnType;
-        }
+        intermediateType = TypeDef.createVarchar(-1);
```

Comment:
use createVarchar(max length of varchar) would be a moderate way. -1 looks so unsafe.